src/test/java/com/revature/studyforce/flashcard/service/RatingServiceTest.java

```diff
@@ -35,31 +36,50 @@
     @MockBean
     private RatingRepository ratingRepository;
     @MockBean
-    private FlashcardRepository flashcardRepo;
+    private FlashcardRepository flashcardRepository;
     @MockBean
     private UserRepository userRepository;
 
     @Autowired
     private RatingService ratingService;
 
     @Test
-    void whenCreateRating_returnsRatingResponseDTO(){
+    void givenRatingDTO_whenCreateRating_shouldReturnRatingResponseDTO(){
         List<Rating> rList = new ArrayList<>();
         User user = new User(0,"edson@revature.com","password","Edson","Rodriguez",true,false,false, Authority.USER, Timestamp.valueOf(LocalDateTime.now()),Timestamp.valueOf(LocalDateTime.now()));
-        Flashcard flashcard = new Flashcard(0,user,null,"how is your day",1,1,Timestamp.valueOf(LocalDateTime.now()),null);
+        Flashcard flashcard = new Flashcard(0,user,null,"how is your day",1,1,Timestamp.valueOf(LocalDateTime.now()),null,false);
         Rating rating = new Rating(0,flashcard,user, Difficulty.EASY);
         rList.add(rating);
 
-        Mockito.when(flashcardRepo.findById(0)).thenReturn(Optional.of(flashcard));
+        Mockito.when(flashcardRepository.findById(0)).thenReturn(Optional.of(flashcard));
         Mockito.when(userRepository.findById(0)).thenReturn(Optional.of(user));
         Mockito.when(ratingRepository.findByFlashcard_id(0)).thenReturn(rList);
         Mockito.when(ratingRepository.save(org.mockito.ArgumentMatchers.isA(Rating.class))).thenReturn(rating);
 
         RatingResponseDTO res = ratingService.createRating(RatingDTO.ratingToDTO().apply(rating));
         assertNotNull(res);
-        assertTrue(res.getRating()==0);
+        assertEquals(1, res.getRating());
         assertTrue(res.getTotalRatings()>0);
 
         System.out.println(res);
     }
+
+    @Test
+    void givenFlashcardIdAndUserId_whenGetRating_shouldReturnRating(){
+        User user = new User(0,"edson@revature.com","password","Edson","Rodriguez",true,false,false, Authority.USER, Timestamp.valueOf(LocalDateTime.now()),Timestamp.valueOf(LocalDateTime.now()));
```

Comment:
Using outdated user model, please correct to user model with 1 name field and no password field